webapp/channels/src/components/post_edit_history/edited_post_item/edited_post_item.tsx

```diff
@@ -194,6 +194,7 @@ const EditedPostItem = ({post, isCurrent = false, postCurrentVersion, theme, act
                 onClick={openRestorePostModal}
                 compact={true}
                 aria-label={formatMessage(itemMessages.ariaLabelMessage)}
+                onMouseDown={(e) => e.stopPropagation()}
```

Comment:
This is currently causing the type checker to fail because the type definitions are missing this field, but I don't know if this is actually doing anything because clicking on the restore button still expands or collapses the row for me